terraform/modules/cloudfoundry/variables.tf

```diff
@@ -145,3 +145,59 @@ variable "waf_label_host_0" {
 variable "waf_hostname_0" {
   type = string
 }
+
+variable "nat_egress_ip_set_arn" {
+  type    = string
+  description = "ARN for IP set of CIDR ranges for NAT Gateways for this environment"
+}
+
+variable "tooling_nat_egress_ip_set_arn" {
+  type    = string
+  description = "ARN for IP set of CIDR ranges for NAT Gateways for the tooling environment"
+}
+
+variable "user_agent_header_name" {
+  type = string
+  description = "Header name to identify user agent"
+  default = "user-agent"
+}
+
+variable "cloudfront_user_agent_header" {
+  type = string
+  description = "User-Agent header value used to identify Cloudfront traffic"
+  default = "Amazon Cloudfront"
+}
+
+variable "forwarded_ip_header_name" {
+  type = string
+  description = "Header name used to get forwarded IP addresses"
+  default = "X-Forwarded-For"
+}
+
+variable "non_cdn_traffic_rate_limit_challenge_by_source_ip" {
+  type = number
+  description = "Number of requests to allow per source IP per 5 minute interval before challenging requests"
+  default = 2000
+}
+
+variable "non_cdn_traffic_rate_limit_challenge_by_forwarded_ip" {
+  type = number
+  description = "Number of requests to allow per forwarded IP per 5 minute interval before challenging requests"
+  default = 2000
+}
+
+variable "non_cdn_traffic_rate_limit_block_by_forwarded_ip" {
+  type = number
+  description = "Number of requests to allow per forwarded IP per 5 minute interval before blocking requests"
+  default = 5000
```

Comment:
This threshold was chosen completely arbitrarily. I figured the thresholding for blocking should be higher for challenging (2000) and I opted to be very conservative, given that our current threshold for blocking was so high it was not doing anything